tools/scripts/create-json-for-manifest.ps1

```diff
@@ -165,6 +165,6 @@ function CreateOutputFile([string]$json, [string]$outputPath, [string]$outputFil
 }
 
 $resolvedOutputPath = Resolve-Path $OutputPath
-$json = CreateJsonForManifest $MsiBasePath $OctokitRelativePath $isMandatoryProdUpdate
```

Comment:
At this point, `$isMandatoryProdUpdate` (lower case 'i') has not been set, so it has no value. I'll add the requirement that would trigger this as an error